packages/common/src/adapters/activity.ts

```diff
@@ -3,7 +3,7 @@ import { full } from '@audius/sdk'
 import { userCollectionMetadataFromSDK } from './collection'
 import { userTrackMetadataFromSDK } from './track'
 
-export const repostActivityFromSDK = (input: full.ActivityFull) => {
+export const activityFromSDK = (input: full.ActivityFull) => {
```

Comment:
I cannot remember if repost and favorite are compatible responses, so this needs some testing 😅